Cargo.toml

```diff
@@ -135,6 +135,7 @@ dpe = { path = "dpe/dpe", default-features = false, features = ["dpe_profile_p38
 crypto = { path = "dpe/crypto", default-features = false }
 platform = { path = "dpe/platform", default-features = false }
 elf = "0.7.2"
+enum-map = "2.7.3"
```

Comment:
I don't think enum-map is being used anymore. Should this be removed here and in other Cargo.toml files?